### PR TITLE
link component

### PR DESCRIPTION
**Pull request by naremanhilles**
Relates #24

### CODE REVIEW

**Review by AbdallahAmmar96 — APPROVED** [review]

File: Client/src/components/common/Link/index.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+
+function Link(props){
```
Comment (kannanhassouna) [inline Client/src/components/common/Link/index.js:3]: Indentation

**Review by kannanhassouna — CHANGES_REQUESTED** [review]
the Style Again

**Comment by naremanhilles** [comment]
style fatma task.

**Comment by Fatmasiam** [comment]
There is conflict ?! solve it

**Comment by kannanhassouna** [comment]
@naremanhilles Solve the conflict

**Pull request closed without merging**